src/sidebar/components/MarkdownEditor.js

```diff
@@ -398,7 +398,7 @@ export default function MarkdownEditor({
   const [preview, setPreview] = useState(false);
 
   // The input element where the user inputs their comment.
-  const input = useRef(/** @type {HTMLTextAreaElement|null} */ (null));
+  const input = /** @type {{ current: HTMLTextAreaElement }} */ (useRef());
```

Comment:
I believe this ref could be null (it is conditionally rendered).

Reply:
Oh indeed, you're right. I'll correct this.